### ISSUE TITLE
Github for Developers - Webnair recording

### ISSUE THREAD

**Issue by RamuKMLG** [githubschool/funny-lemur#157]
Hi,

Any possibility to get this recording to view later?
If so, I will be glad to review and use it later.

/Ramu

**Comment by crichID** [comment]
@RamuKMLG yes, we will send you a link to the recording via email after class today!

**Comment by RamuKMLG** [comment]
Great!
Thank you!!

**Comment by sfsofia** [comment]
Hi ,
Please send this records to me also.
I need this badly, My laptop was not helping me much ,I had to do a restart.

**Comment by crichID** [comment]
@RamuKMLG @sfsofia can you please email me at services@github.com and I will get you the link!
